src/jace/util/jax_helper.py

```diff
@@ -258,3 +264,42 @@ def get_jax_literal_value(lit: jax_core.Atom) -> bool | float | int | np.generic
     if util.is_scalar(val):
         return val
     raise TypeError(f"Failed to extract value from '{lit}' ('{val}' type: {type(val).__name__}).")
+
+
+def parse_backend_jit_option(
+    backend: str | dace.DeviceType,
+) -> dace.DeviceType:
+    """Turn JAX' `backend` option into the proper DaCe device type."""
+    if isinstance(backend, dace.DeviceType):
+        return backend
+    match backend:
+        case "cpu" | "CPU":
+            return dace.DeviceType.CPU
+        case "gpu" | "GPU":
+            return dace.DeviceType.GPU
+        case "fpga" | "FPGA":
+            return dace.DeviceType.FPGA
+        case "tpu" | "TPU":
+            raise NotImplementedError("TPU are not supported.")
```

Comment:
I don't see why this is useful if it is not supported

Reply:
It is a better error message.
I think the message that JaCe does not support TPU is much better than an error that is indicating that the parsing failed.

Reply:
This function should accept only strings or it should be renamed, because right now is not a parsing function but a `dace.DeviceType` maker, and that's why the "Could not parse backend" message is weird. You could just write something like "'{backend}' is not a valid DaCe device type name"